src/cpp/fastrtps/rtps/builtin/discovery/participant/PublicationMatchedStatus.h

```diff
@@ -0,0 +1,55 @@
+// Copyright 2019 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @file PublicationMatchedStatus.h
+*/
+
+#ifndef _PUBLICATION_MATCHED_STATUS_H_
+#define _PUBLICATION_MATCHED_STATUS_H_
+
+#include <cstdint>
+#include <fastrtps/rtps/common/InstanceHandle.h>
+#include <fastrtps/rtps/common/MatchingInfo.h>
+
+namespace eprosima{
+namespace fastrtps{
+
+//! @brief A structure storing the publication status
+struct PublicationMatchedStatus
+{
+	//! @brief Total cumulative count the concerned writer discovered a match with a reader
+	//! @details It found a reader for the same topic with a requested Qos that is compatible with that offered by the writer
+	uint32_t total_count;
```

Comment:
It would be better to use int32_t instead as RTI does.

Reply:
Changed